rotors_gazebo_plugins/include/rotors_gazebo_plugins/common.h

```diff
@@ -23,9 +23,45 @@
 
 #include <Eigen/Dense>
 #include <gazebo/gazebo.hh>
+#include <mav_msgs/default_topics.h>
 
 namespace gazebo {
 
+// Default values
+static const std::string kDefaultNamespace = "";
+
+static constexpr double kDefaultRotorVelocitySlowdownSim = 10.0;
+
+static const std::string kDefaultGroundTruthPoseTopic =
+    mav_msgs::default_topics::GROUND_TRUTH_POSE; // "ground_truth/pose";
```

Comment:
these are all in `mav_msgs::default_topics::`now, I guess we can remove the constants here.